contrib/!lang/ruby/packages.el

```diff
@@ -32,6 +32,14 @@
     :config (add-hook 'enh-ruby-mode-hook
                       (lambda () (rbenv-use-corresponding)))))
 
+(defun ruby/init-chruby ()
+  "Initialize chruby mode"
+  (use-package chruby
+    :defer t
+    :init (chruby)
+    :config (add-hook 'enh-ruby-mode-hook
```

Comment:
You have to replace `:config` by `:init` otherwise the function won't be hooked at initialization time (ie before the actual loading of the package).